index.html

```diff
@@ -39,77 +39,312 @@
+              <br />
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+          <!-- About section -->
+          <section class="about">
+            <section class="abt-main">
+              <br />
+              <h3>About me</h3>
               <br />
               <p>
-                My name is Dagmawi Yohannes I'm a Front End Engineer based in
-                Addis Ababa, Ethiopia ☀️. I describe myself as a passionate
-                developer who loves coding, open source, and the web platform
+                I can help you build a product,
+                feature or website Look through
+                some of my work and experience! If 
+                you like what you see and have a 
+                project to be coded,
+                don't hesitate to Contact me.
               </p>
               <br />
-              <div class="h2"></div>
-              <div class="social-links">
-                <ul class="ulist">
-                  <li class="ulist-icon">
-                    <a href="https://twitter.com/dag_mawi111 "
-                      ><img src="./images/twitter.svg" alt="twitter icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
-                      ><img src="./images/Linkedinicon.svg" alt="linkedin icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://medium.com/dagmawi.yoha03"
-                      ><img src="./images/medium.svg" alt="medium icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://github.com/dagmawiii03"
-                      ><img src="./images/github.svg" alt="github icon"
-                    /></a>
-                  </li>
-                </ul>
-              </div>
+              <button>Get My Resume</button>
             </section>
-            <section class="pointer">
+          <section class="abt-card">
+            <section class="card">
+              <br />
+              <img src="./images/languages-icon.png" alt="" />
+              <br />
+              <h1>Languages</h1>
+              <br />
               <ul>
-                <li>
-                  <a href=""><i class="fas fa-chevron-down white-color"></i></a>
-                </li>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>CSS</li>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design

  - Please the margin in between the skill icon, the text, and the skill buttons is so much and does not look the same as the Figma design. 

| Your Design |Figma Design |
| :---:   | :---: | 
|![image](https://user-images.githubusercontent.com/92303891/219874283-016f44df-f848-4e7f-bfd2-470719c33a4d.png) | ![image](https://user-images.githubusercontent.com/92303891/219874406-3d0a4606-56a1-49c9-a809-e4b132e99efd.png) |

Reply:
Done